README.md

```diff
@@ -69,19 +69,16 @@ process.mainArgs = process.argv.slice(process._exec ? 1 : 2)
 
 ----
 
-## 💡 `util.parseArgs([argv][, options])` Proposal
+## 💡 `util.parseArgs([config])` Proposal
 
-* `argv` {string[]} (Optional) Array of argument strings; defaults
-  to [`process.mainArgs`](process_argv)
-* `options` {Object} (Optional) The `options` parameter is an
+* `config` {Object} (Optional) The `config` parameter is an
```

Comment:
Any interest in representing this as a type alias? This is an extremely biased opinion, but this is much more readable to me. Understanding, this could be a barrier for folks not familiar with `TypeScript` conventions (such as: `?` optional object properties)
e.g.
```ts
/**
  * `parseArgs` configuration options
  */
type config = {
  /**
   * Array of argument strings
   * @default process.mainArgs
   */
  argv?: string[];
  /**
   * A collection of configuration objects for each `argv`
   */
  options?: {
    [longName: string]: {
      /**
        * Type of `argv`
        * @default 'boolean'
        */
      type?: 'string' | 'boolean'; 
      /**
       * If true, when appearing multiple times in `argv`, will be concatenated into an Array
       * @default false
       */
      multiples?: boolean;
      /**
       * An alias to an `argv`; When appearing multiples times in `argv`; Respects the `multiples` configuration
       */
      short?: string; 
    };
  };
  /**
    * A Boolean on whether or not to throw an error when unknown args are encountered
    */
  strict?: boolean;
}



```

Reply:
Scratch this.. I'm not seeing this convention used anywhere in existing Node documentation.